test/run-tests.js

```diff
@@ -0,0 +1,46 @@
+import { spawn } from 'child_process';
```

Comment:
please rename the file to .mjs

Reply:
same below

Reply:
updated with commit https://github.com/appium/appium-ios-tuntap/pull/1/commits/c4616b912841f07ca4cce23afad54e996759fc2f